storefront-reference-architecture-master/cartridges/int_twilio/cartridge/controllers/PhoneNumber.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+/**
+ * @namespace PhoneNumber
+ */
```

Comment:
It can be added within the Account controller

Reply:
moved it